Entity/Concrete/Category.cs

```diff
@@ -1,20 +1,39 @@
-﻿using WebNetSample.Core.Entities;
+﻿using Microsoft.AspNetCore.Http;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using WebNetSample.Core.Entities;
 
 namespace WebNetSample.Entity.Concrete;
 
 public class Category : BaseEntity
 {
+    [DisplayName("Category Name")]
+    [Required(AllowEmptyStrings = false, ErrorMessage = "Please enter a name")]
+    [StringLength(maximumLength: 50, MinimumLength = 3, ErrorMessage = "Length must be between 3 to 50")]
     public string Name { get; set; }
 
+    [DisplayName("Category Image")]
+    [Required(AllowEmptyStrings = false, ErrorMessage = "Please enter an image path")]
+    public string ImagePath { get; set; }
+
+    [NotMapped]
+    public IFormFile? ImageFile { get; set; }
```

Comment:
No need to use `IFormFile` in entity, entity should store **only** the path to your file, not its content. Please, move `IFormFile` to the DTO